modules/redis-cluster/main.tf

```diff
@@ -32,6 +32,13 @@ resource "google_redis_cluster" "redis_cluster" {
     }
   }
 
+  dynamic "zone_distribution_config" {
+    for_each = var.zone_distribution_config_mode != "" ? [1] : []
+    content {
+      mode = var.zone_distribution_config_mode
+    }
+  }
+
```

Comment:
@imrannayer `zone_distribution_config` has been updated to dynamic block, please review again. Thanks

Reply:
@lanandra `zone_distribution_config` has two properties `mode` and `zone`. Can you plz add `zone` to it?